README.md

```diff
@@ -1 +1,53 @@
-# TodoList
\ No newline at end of file
+# ToDo List App
+
+Todo List is a web application that allows users to add/remove the list of "to-dos". It provides an interface for adding to do activities, for visualizing the list of added tasks to complete. This is achieved using plain HTML, CSS, JavaScript objects, arrays, DOM, basic events, and webpack.
+
+![Screenshot](src/Screenshot.png)
```

Comment:
- [ ]  OPTIONAL
This _screenshot_ link is broken. Kindly endeavor to fix it

Reply:
Ok, thank you so much